decidim-admin/app/permissions/decidim/admin/permissions.rb

```diff
@@ -21,6 +21,7 @@ def permissions
         allow! if user_can_enter_space_area?
 
         read_admin_dashboard_action?
+        admin_newsletter_action?
```

Comment:
Why you think that it has side effects? the `admin_newsletter_action?` was inspired in the above method `read_admin_dashboard_action?` that returns an `allow!` or `disallow!` as most of the methods in Decidim::Permissions

It could be refatored it to return a `true/false` and then use a conditional `allow! if user_can_manage_newsletter?`

About the renaming, maybe `newsletter_action?` is more appropiate as seen in `decidim-core/app/permissions/decidim/permissions.rb`, I think that prepending `apply_permissions_` to the method doesn't make sense as the rest of them are also applying permissions and don't use this nomenclature, having in mind that its in the Permissions class.

https://github.com/decidim/decidim/blob/master/decidim-core/app/permissions/decidim/permissions.rb